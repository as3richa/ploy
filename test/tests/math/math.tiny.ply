(/ (* 1796979272) (+ (*) (+ 2125812833 699284889) 304485310) 527715646)
(+ 1471947512 (+ 633843034 880179185) 1367581832)
2929425374
(*)
4084883705
551155618
(* (- (+) (/ 1277239443) (/ 3253159220)))
3139472388
(+ (* (/ 2121735993)))
(*)
