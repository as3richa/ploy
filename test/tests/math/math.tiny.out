(/ 898489636 825764965985117159)
4353551563
2929425374
1
4084883705
551155618
(/ -4530398663 4155063270143114460)
3139472388
(/ 1 2121735993)
1
